src/Explorer/Tree/ResourceTree.tsx

```diff
@@ -397,6 +397,11 @@ export const ResourceTree: React.FC<ResourceTreeProps> = ({ container }: Resourc
       },
     ];
 
+    // For Temporary workspaces removing "Rename" in context menu.
```

Comment:
>context [](http://example.com/codeflow?start=53&length=7)

why are we disabling rename for temp workspace?

Reply:
Updated description in the bug.